app/(route)/ranking/page.tsx

```diff
@@ -1,32 +1,68 @@
 'use client';
 
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Button from '@/components/common/Button';
 import ItemBoxWithoutLike from '@/components/ShopPage/ItemBoxWithoutLike';
+import axios from 'axios';
 
-const RankingPage = () => {
+interface RankingProduct {
+    id: number;
+    productCode: string;
+    productName: string;
+    productSubName: string;
+    price: string;
+    productImageResponse: {
+        productImage: string[];
+    };
+}
+
+const RankingPage: React.FC = () => {
```

Comment:
```suggestion
const RankingPage = () => {
```
불필요한 타입 정의 삭제하고 커밋해주세요!

그리고 저희가 props 타입 정의할때 React.FC<>랑 {}:Props를 혼용하고 있는데 나중에 하나로 정하는게 좋을 것 같습니다.